data/shell-completions/bash/meson

```diff
@@ -195,6 +195,7 @@ _meson-setup() {
     sysconfdir
```

Comment:
I wonder if there is some refactor here that could save us from duplicating everything